src/plugins/intel_npu/cmake/features.cmake

```diff
@@ -4,12 +4,22 @@
 
 ov_option(ENABLE_MLIR_COMPILER "Enable compilation of npu_mlir_compiler libraries" ON)
 
-ov_option(BUILD_COMPILER_FOR_DRIVER "Enable build of npu_driver_compiler" OFF)
+ov_option(ENABLE_NPU_RUNTIME_COMMON "Enable compilation of npu runtime common libraries" ON)
```

Comment:
Why do we need this option? It will not be built if the target is not used.

Reply:
We would like to remove BUILD_COMPILER_FOR_DRIVER from the openvino project. We will expose all necessary flags to allow other projects do disable unwanted targets. We reviewed with @XinWangIntel the complexity of achieving the same without exposing these options and it is not acceptable at the moment.